tests/integration/servicemesh/federation/federation_test.go

```diff
@@ -0,0 +1,62 @@
+//go:build integ
+// +build integ
+
+// Copyright Red Hat, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package servicemesh
+
+import (
+	"testing"
+
+	"istio.io/istio/pkg/test/framework"
+	"istio.io/istio/pkg/test/framework/components/istio"
+	"istio.io/istio/pkg/test/framework/label"
+	"istio.io/istio/tests/integration/servicemesh/maistra"
+)
+
+var i istio.Instance
+
+// GetIstioInstance gets Istio instance.
+func GetIstioInstance() *istio.Instance {
+	return &i
+}
+
+func TestMain(m *testing.M) {
+	// Ignoring staticcheck linter. RequireMinClusters is now deprecated, but we need at least 2 clusters for this
+	// test to make it work.
+	// nolint: staticcheck
+	framework.
+		NewSuite(m).
+		RequireMinClusters(2).
+		Setup(maistra.ApplyServiceMeshCRDs).
+		Setup(istio.Setup(GetIstioInstance(), setupConfig)).
+		Run()
+}
+
+func TestFederation(t *testing.T) {
+	framework.NewTest(t).
+		Label(label.Multicluster).
```

Comment:
let's remove the label. this is now covered by you adding the RequireMaxClusters(1) directive in the other tests. but we should first remove the reference from the [prow job](https://github.com/maistra/test-infra/blob/29a72e3a2cfd49d189f7e66be3fd399c8ab40d9f/prow/config/presubmits.yaml#L1035)

Reply:
As a matter of fact, we already use the non-existing label in this job - `-flaky`. We can have those two changes in parallel.

Reply:
> this is now covered by you adding the RequireMaxClusters(1) directive in the other tests

That's not entirely true. In such case, it would try running `federation` tests again single-cluster setup. We do have, however, another directive requesting more than 1 cluster for such a  test. This makes the label in fact redundant.